prow/config/config.go

```diff
@@ -38,6 +38,15 @@ type Config struct {
 
 	Plank    Plank     `json:"plank,omitempty"`
 	Triggers []Trigger `json:"triggers,omitempty"`
+
+	// ProwJobNamespace is the namespace in the cluster that prow
+	// components will use for looking up ProwJobs. The namespace
+	// needs to exist and will not be created by prow.
+	ProwJobNamespace string `json:"prowjob_namespace,omitempty"`
```

Comment:
This should be fine, at least for now. I'm not sure what the config will look like in the future, since some of it will live in the repos under test.